build.gradle

```diff
@@ -97,6 +97,8 @@ ext {
     awsCommand = "aws"
 
     sdkZip = file("${buildDir}/java-sdk-${version}.zip")
+
+    swaggerVersion = "1.5.16"
```

Comment:
Oh, hm, didn't know about that. Looks like it may take some thinking to integrate that, and I'm pretty unfamiliar with it. Any objections to coming back to retrofit that? This is currently breaking all builds.